oauth2/src/main/java/com/okta/spring/boot/oauth/OktaOAuth2ResourceServerAutoConfig.java

```diff
@@ -64,15 +70,37 @@ JwtDecoder jwtDecoder(OAuth2ResourceServerProperties oAuth2ResourceServerPropert
                           OktaOAuth2Properties oktaOAuth2Properties) {
 
         NimbusJwtDecoder.JwkSetUriJwtDecoderBuilder builder = NimbusJwtDecoder.withJwkSetUri(oAuth2ResourceServerProperties.getJwt().getJwkSetUri());
-        builder.restOperations(restTemplate());
+        builder.restOperations(restTemplate(oktaOAuth2Properties));
         NimbusJwtDecoder decoder = builder.build();
         decoder.setJwtValidator(TokenUtil.jwtValidator(oktaOAuth2Properties.getIssuer(), oktaOAuth2Properties.getAudience()));
         return decoder;
     }
 
-    private RestTemplate restTemplate() {
+    private RestTemplate restTemplate(OktaOAuth2Properties oktaOAuth2Properties) {
```

Comment:
Remove the duplicate code from: `OktaOidcUserService`

Reply:
Done!